tools/mitohifi/mitohifi.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="mitohifi" name="MitoHiFi" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="2.9.0">blast</requirement>
+        <requirement type="package" version="1.78">biopython</requirement>
+        <requirement type="package" version="1.2.2">pandas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        bash '$__tool_directory__/run.sh' -c '$contigs' -f '$mitogenome_fa' -g '$mitogenome_gb' -t \${GALAXY_SLOTS:-1} -o $organism
+    ]]></command>
+    <inputs>
+        <param type="data" name="contigs" format="fasta" label="" />
+        <param type="data" name="mitogenome_fa" format="fasta"  label="Closely-related species mitogenome in fasta format" />
```

Comment:
Are both files needed or just one of them?

Reply:
closely vs. close-releated

Reply:
I thought "closely-" was grammatically correct, but now I see there are papers that use this wording in their title - so I was wrong.

Reply:
> Are both files needed or just one of them?

As far as I can tell, both. As per README, and what run.sh does.